docker-images/cold-backup/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.13
```

Comment:
This should use the standard base image - either the strimzi base image or CentOS 7.

Reply:
No problem with this, it just seems an overkill for `rsync` only. Given that you are doing a backup, the base image is already cached, nevertheless Alpine based images are lighter and, most importantly, quicker at runtime.